perf/server.py

```diff
@@ -60,7 +60,7 @@ def remote_get_free_cores_ssh(self, threads):
     proc = os.popen(ssh_cmd_str)
     result = proc.read().strip()
     if len(result)==0:
-      return "(False, 0, 0, 0)"
+      return "(False, 0, 0, 0, 0)"
     result = ast.literal_eval(result)
     # (free, mem, start, end, server_cores)
```

Comment:
Should we update this line of comments accordingly?

Reply:
The format of the return value is commented as line 65, which is here to fix the default quadruple to the default quintuple.